resources/energyplus/ProposedEnergy+.idd

```diff
@@ -5118,6 +5118,36 @@ MaterialProperty:VariableThermalConductivity,
        \note for Temperature-Thermal Conductivity function corresponding to temperature 1
        \units W/m-K
 
+MaterialProperty:VariableAbsorptance,
```

Comment:
Sneaking this in (unrelated to this PR), right?

Reply:
Yes. I realized between this PR and the ones you added so far, it was going to be the only missing one.